A/ADIOS2/build_tarballs.jl

```diff
@@ -123,13 +164,9 @@ platforms = expand_cxxstring_abis(platforms)
 # <https://github.com/ornladios/ADIOS2/issues/2704>
 filter!(p -> nbits(p) ≠ 32, platforms)
 
-# riscv64 builds fail.
-# It seems that some vendored libraries aren't built, leading to undefined references later on
-# ld: warning: libadios2_perfstubs.so.2.10, needed by lib/libadios2_core_mpi.so.2.10.2, not found (try using -rpath or -rpath-link)
-# ld: warning: libadios2_evpath.so.2.10, needed by lib/libadios2_core.so.2.10.2, not found (try using -rpath or -rpath-link)
-# ld: warning: libadios2_ffs.so.2.10, needed by lib/libadios2_core.so.2.10.2, not found (try using -rpath or -rpath-link)
-# ld: warning: libadios2_atl.so.2.10, needed by lib/libadios2_core.so.2.10.2, not found (try using -rpath or -rpath-link)
-filter!(p -> arch(p) != "riscv64", platforms)
+# There are build errors on Windows, probably fixable
+# toolkit/transport/file/FilePOSIX.cpp:448:35: error: ‘pread’ was not declared in this scope
```

Comment:
Feels like an upstream bug if this is entering a file called `FilePOSIX` when targeting Windows.